### ISSUE TITLE
Public Keys > RMD160

### ISSUE THREAD

**Issue by sezginyildirim91** [albertobsd/keysubtracter#14]
Hello Alberto. I want to convert Public Keys to RIPEMD-160 but I couldn't succeed. I've been struggling for weeks, please guide me.

```
if(mpz_tstbit(dst_publickey.y, 0) == 0)	{	// Even
	gmp_fprintf(OUTPUT,"02%0.64Zx\n",dst_publickey.x,sum_key);
	RMD160Data((const unsigned char*)dst_publickey.x,32, digest160);
```

**Comment by albertobsd** [comment]
I will add that opcion, this will be 3 types of output 

- publickeys (default)
- rmd160
- address

and maybe compress, uncompress or both but im not sure how usefull it can be.

**Comment by albertobsd** [comment]
Last commit solved this 3c575468ae77cd541d4837019ef79a735cab5070
